src/pages/Team/Team.tsx

```diff
@@ -6,15 +6,29 @@ import PersonCard from './components/PersonCard';
 
 const Team: React.FC = () => {
   return (
-    <Page className={styles.team} contentClassName={styles.content}>
-      {team.map((person) => (
+    <Page
+      animation="slide"
+      className={styles.team}
+      contentClassName={styles.content}
+    >
+      <div className={styles.header}>
+        <span className={styles.adooTxt}>Adooone</span>
+        &nbsp;
+        <span className={styles.teamTxt}>Team</span>
+      </div>
+      {team.map((person, index) => (
         <PersonCard
+          right={index % 2 === 1}
+          avRight={index % 2 === 1}
+          index={index}
           key={person.id}
           bio={person.bio}
           name={person.name}
           position={person.position}
+          avatar={person.avatar}
         />
       ))}
+      <div className={styles.footer}>Every little thing gonna be allright.</div>
```

Comment:
Это ж текст, нужен тег `p`
```suggestion
      <p className={styles.footer}>Every little thing gonna be allright.</p>
```